api/HID.json

```diff
@@ -39,6 +39,51 @@
           "deprecated": false
         }
       },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "131",
+                "partial_implementation": true,
+                "notes": "Dedicated workers and WebExtension service workers only."
+              },
+              {
+                "version_added": "117",
+                "partial_implementation": true,
+                "notes": "WebExtension service workers only."
+              }
```

Comment:
Should we document where it works (like here), or where it doesn't work (like in [this PR](https://github.com/mdn/browser-compat-data/pull/24639))?

From a user point of view, it might be more interesting to know where it _can_ be used, but I feel like it's easier to track feature _gaps_ or limitations, i.e. where it cannot be used.

Thoughts?

Reply:
I personally think it is better to document where things _do_ work, as then it sounds clearer and is less likely to leave the reader guessing — "Hrm, well it isn't supported in those places, but where is it supported?"

However, these two cases are different — the linked PR is stating overall support, rather than just support in workers. If you said where things are supported, then you'd have to change it to "Available in Window and dedicated workers only." or similar, which might equally confuse some people.

I don't know which is best, there is no clear winner. I think we should choose one and stick to it.

Reply:
What is the recommended approach here, @queengooborg? Document what is supported, or what isn't?

Reply:
@chrisdavidmills We discussed this in today's BCD call, and aligned on a preference for precise and comprehensive notes on what works and what doesn't work.

Reply:
So...what does that mean, exactly?

Reply:
Sorry, missed to give an example: Here you could specify that the feature is available in worker types X and Y, but it is not available in worker type Z. This makes the availability explicit, and if the spec changes to include another worker type, it won't be ambiguous.

Reply:
@caugner OK, thanks! I've updated the notes based on the latest guidance. Let me know what you think.

Reply:
LGTM, I actually like it more being this explicit. 🙂

Reply:
Yeah, me too. Thanks for working with me on this!